docker/keycloak/config/keycloak_import.json

```diff
@@ -1181,7 +1181,7 @@
         "forwardParameters": "pres_req_conf_id",
         "tokenUrl": "http://host.docker.internal:5201/token",
         "authorizationUrl": "http://localhost:5201/authorize",
-        "clientAuthMethod": "client_secret_post",
+        "clientAuthMethod": "client_secret_basic",
```

Comment:
Was this meant to be a change? I *think* the default behaviour is to send the secret as part of the `post`.

Either way, this is something we will need to flesh out as we work through the client authentication piece.

Reply:
Yes, the library wanted it that certain way by default, but I'll take another look to see if the library's behaviour is configurable.

Reply:
Sounds good, will look at it as part of the next changes then.